core/lib/types/Coproduct.agda

```diff
@@ -154,3 +154,21 @@ module _ {i j k} {A : Type i} {B : Type j} (P : A ⊔ B → Type k) where
       from-to : ∀ fg → from (to fg) == fg
       from-to fg = λ= λ where (inl _) → idp
                               (inr _) → idp
+
+-- the empty type is a unit for the coproduct
+Coprod-unit-left : ∀ {i} (A : Type i) → ⊥ ⊔ A ≃ A
+Coprod-unit-left {i} (A) = equiv to from to-from from-to
```

Comment:
`unit-l` (instead of `unit-left`) seems to be more coherent with what have been done for groups. Would you mind changing it?